client/cgo/cgotest.go

```diff
@@ -1,7 +1,13 @@
 package main
 
 /*
-int testVerify(int type,
+struct cgoVerify_return {
+	int r0;
+	int r1;
+};
+extern struct cgoVerify_return cgoVerify(int p0, char* p1, int p2, void* p3, int p4, long long unsigned int p5, void* p6, int p7, void* p8, int p9, char* p10, int p11);
```

Comment:
Also, you seem to be using `long long unsigned int` for `p5` here, but `unsigned long long` for `currentEpoch` below. Is this intentional?